File: Samples/JotFormTest/Classes/Controller/LoadSettingsViewController.m
//
//  LoadSettingsViewController.m
//  JotFormTest
//
//  Created by Curtis Stilwell on 8/21/17.
//  Copyright © 2017 wang. All rights reserved.
//

#import "LoadSettingsViewController.h"
#import "SharedData.h"

@interface LoadSettingsViewController ()

@property (nonatomic, weak) IBOutlet UITextView *textView;

@end

@implementation LoadSettingsViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view from its nib.
    [[SharedData sharedData].apiClient getSettings:^(id result) {
        
        self.textView.text = [NSString stringWithFormat:@"%@",result];
        
    } onFailure:^(NSError *error) {
        
    }];
}

@end

File: Samples/JotFormTest/Classes/Controller/LoadSettingsViewController.swift
//
//  LoadSettingsViewController.swift
//  JotFormTest
//
//  Created by Curtis Stilwell on 2/4/18.
//  Copyright © 2018 wang. All rights reserved.
//

import Foundation

class LoadSettingsViewController: UIViewController {
    
    @IBOutlet weak var textView: UITextView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view from its nib.
        
        title = "Load Settings"
        loadSettings()
    }
    
    func loadSettings() {
        SharedData.sharedData.apiClient?.getSettings({(_ result: Any) -> Void in
            self.textView.text = "\(result)"
        }, onFailure: {(_ error: Error?) -> Void in
        })
    }
}
